### DISCUSSION TITLE
Problema en el login

### DISCUSSION THREAD

**Discussion by Angierm12** [rramosp/ai4eng.v1#221] [01 - INTRODUCCIÓN]
Buenos días, he intentado ingresar por medio del correo y la contraseña y también con el login de Google, pero me sale el mensaje de que no estoy registrada en ninguna sesión del curso y en la celda de ejecución para enviar la solución en línea me sale el error de la segunda imagen.
<img width="943" alt="Problema en el login" src="https://user-images.githubusercontent.com/124899252/218769472-ddd3af91-a592-444d-9755-d9bb1395c797.png">
<img width="645" alt="Problema en el login 2" src="https://user-images.githubusercontent.com/124899252/218769497-9574bde6-afe2-40ef-831f-a61dd0e1fafd.png">

**Comment by Felipe-RA** [comment]
Saludos @Angierm12 

¿Ya hiciste el registro por el enlace en la página principal del curso?

**Reply by Angierm12** [reply]
Hola, 
Sí, yo ya me había registrado en la plataforma, pero no había aceptado la invitación al curso. Ya lo solucioné. Muchas gracias.
